### PR TITLE
Unused TCP port factory fixture. Bump to 0.2.0.

### PR DESCRIPTION
**Pull request by Tinche**
I've decided to simplify the new fixture a little; now it's a callable which returns a different port each invocation.

@nicoddemus Could you please do a small review before merging?

### CODE REVIEW

File: pytest_asyncio/plugin.py [hunk 1]
```diff
@@ -88,3 +88,17 @@ def unused_tcp_port():
...
+
+@pytest.fixture
+def unused_tcp_port_factory():
+    """A factory function, producing different unused TCP ports."""
+    produced = set()
+
+    def factory():
+        port = unused_tcp_port()
+        while port in produced:
+            port = unused_tcp_port
```
Comment (nicoddemus) [inline pytest_asyncio/plugin.py]: I think this should be `port = unused_tcp_port()`.

**Comment by nicoddemus** [comment]
Hmm coverage has decresed by 2%, but I can't access the source when I try to access coveralls...

File: pytest_asyncio/plugin.py (hunk 1 above)
Comment (Tinche) [inline pytest_asyncio/plugin.py]: Good catch! Of course the bug is in the section not tested. I'll add a test and get coverage up to 100%.

**Comment by nicoddemus** [comment]
Looks good to me, nice work! :smile:

**Comment by nicoddemus** [comment]
(Not sure if I was supposed to merge it)

**Pull request merged**